hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnector.java

```diff
@@ -279,7 +292,7 @@ public Vertex fullScanReader(
     public VertexWithInputConfig insertProcessor(@Nonnull DagBuildContext context) {
         JdbcTable table = (JdbcTable) context.getTable();
 
-        InsertQueryBuilder builder = new InsertQueryBuilder(table.getExternalName(), table.dbFieldNames());
+        InsertQueryBuilder builder = new InsertQueryBuilder(table.getExternalName()[0], table.dbFieldNames());
```

Comment:
`MySQLUpsertQueryBuilder` and others support quoting

Reply:
#24201

Reply:
Addressed in a followup PR

https://github.com/hazelcast/hazelcast/pull/24201/files#diff-047febf2a6668bfd03f71521836eef003be1410622f06c1bfb439c5b60c9e6fbR28-R50